### ISSUE TITLE
Start training from scratch on a new data set, without calling pre-training weights.

### ISSUE THREAD

**Issue by lwzhaojun** [lukemelas/EfficientNet-PyTorch#249]
Hello, thank you very much for your code.
I want to start from scratch on the new data set without calling the pre-training weights.
May I ask what operation is required?
Is it ok to modify the data set address directly in examples/imagenet/main.py?

**Comment by AndrejHafner** [comment]
I'm also interested in fine tuning this network. I ran the `main.py` in `examples/imagenet` on my own dataset, and it starts training, but loss is not decreasing.
